### ISSUE TITLE
Allow "Delivery Timeout" to be disabled

### ISSUE THREAD

**Issue by scottrosenquist** [Yousei9/Logistic-Train-Network#317]
**TL;DR**
I'd like to be able to disable the Delivery Timeout feature, fully aware that this could lead to issues, and that I accept that and will resolve any "lost" trains myself.

**What ?**
I'd like to be able to disable the Delivery Timeout feature.

**Why ?**
I'd like to be able to use my trains _as_ my buffer chests. I'd like to have a train sitting at a station, unloading to belts. I don't want to worry about how long it takes. If a train is unloading to an assembler line that builds rockets, but I haven't used rockets in 24 hours, I want that train to keep sitting there, "delivering." I would use the contents of that train to track and trigger the threshold and request of that LTN requester station so that I can ask for the replacement train at any percentage I want, which I'm already currently able to do with circuits.

I'm sure many people won't care about this feature, I understand that. But some people might, and it's got to be a very simple feature to implement. Throw enough warnings on it that "it's your responsibility to fix your train issues when they come up" and I don't think anyone will accidentally turn it on when they don't want to.

Thanks, and thank you for making LTN in the first place.

**Comment by NightOps007** [comment]
Maybe don't disable the timeout feature, but make it so that ltn won't send a new train while a station is physically occupied. I've seen my trains pile up on one station because of this. You can just disable alerts for that station.

**Comment by DannyParker0001** [comment]
Hello, the way I have my unloaders designed means that trains need to be buffered to sustain full throughput. If this part of the base in inactive for longer than the max timeout time then LTN assume all trains are lost and will re-send all the buffer trains and lock up my train network since there's no space for them. I could continue to make the chest buffers larger, but this is worse for performance I don't see why LTN can't just handle trains being buffered.

Off the top of my head, I can only think of 5 unexpected ways that a delivery could be 'timed out'
- Train gets destroyed
- Station gets destroyed
- Player manually changes trains orders
- No path for a train to get to a station
- Train stuck in deadlock

Assuming I didn't miss anything, if the timeout was set to 0, the only issue that could cause an 'invisible pileup' would be if the train was destroyed. Otherwise, LTN should only send a limited number of trains that shouldn't be hard to find if they get stuck.
Am I missing something here?
